flow-plugins/flow-plugin-base/src/main/java/com/vaadin/flow/plugin/base/ConvertPolymerCommand.java

```diff
@@ -152,11 +156,31 @@ private List<Path> getFilePathsByGlob(Path baseDir, String glob)
         PathMatcher matcher = FileSystems.getDefault()
                 .getPathMatcher("glob:" + glob);
 
-        try (Stream<Path> walk = Files.walk(baseDir)) {
-            return walk.filter(path -> matcher.matches(path))
-                    .filter(path -> !path.toString().contains("node_modules"))
-                    .collect(Collectors.toList());
-        }
+        List<Path> matchingPaths = new ArrayList<>();
+
+        Files.walkFileTree(baseDir, new SimpleFileVisitor<Path>() {
+            @Override
+            public FileVisitResult preVisitDirectory(Path dir,
+                    BasicFileAttributes attrs) throws IOException {
+                if (dir.toString().contains("node_modules")) {
+                    return FileVisitResult.SKIP_SUBTREE;
+                }
+
+                return FileVisitResult.CONTINUE;
+            }
+
+            @Override
+            public FileVisitResult visitFile(Path file,
+                    BasicFileAttributes attrs) throws IOException {
```

Comment:
I believe IOException will never thrown:
```suggestion
                    BasicFileAttributes attrs)  {
```

Reply:
`IOException` is defined by the interface

Reply:
True, my Intellij let me deleted it, so I think it is semantically correct, also could make sense that the implementation is less restrictive than the interface itself. But I am fine both ways!